src/main/java/com/example/librarymanagementsystem/utils/DatabaseConnection.java

```diff
@@ -0,0 +1,21 @@
+package com.example.librarymanagementsystem.utils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnection {
+
+    private static final String URL = "jdbc:postgresql://localhost:5432/libraryManagementSys";
+    private static final String USERNAME = "postgres ";
+    private static final String PASSWORD = "postAdmin";
+
+    public static Connection getConnection(){
+        try{
+            return DriverManager.getConnection(URL,USERNAME,PASSWORD);
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
```

Comment:
Stack traces are used for development purposes only, don't push that to live environments.
Handle the exception that throws in the catch block instead of returning `null`